jbpm-console-ng-generic/jbpm-console-ng-generic-client/src/main/java/org/jbpm/console/ng/gc/client/menu/ServerTemplateSelectorViewImpl.java

```diff
@@ -65,6 +65,23 @@ public void init() {
     }
 
     @Override
+    public void selectServerTemplate(final String serverTemplateId) {
+        if (changeCommand != null) {
```

Comment:
I think you could try to somehow share this code with the AnchorListItem clickhandler from addServerTemplate, they are basically the same once you find the AnchorListItem you need.